src/components/RubricsIndex.js

```diff
@@ -50,7 +51,8 @@ class RubricsIndex extends Component {
                   </tr>
                 </thead>
                 <tbody>
-                  {this.drawRubrics}
+                  {console.log('Props', this.props)}
```

Comment:
Do we need the console.log here?

Reply:
No console.logs on the frontend master.  No exceptions